src/main/java/liuni/health/LiUniHealthCheck.java

```diff
@@ -3,11 +3,8 @@
 import com.codahale.metrics.health.HealthCheck;
 
 public class LiUniHealthCheck extends HealthCheck {
-
-    public LiUniHealthCheck() { }
-
     @Override
-    protected Result check() {
+    public Result check() {
```

Comment:
why the change to public?

Reply:
Originally because I was testing Health check, but then my Health check tests apparently didn't do anything and since they were not on your list I removed those tests.